_posts/2014-03-04-companies.html

```diff
@@ -52,6 +57,11 @@ <h3 id="resources">Resources</h3>
       <td>Company ID (read only)</td>
     </tr>
     <tr>
+      <td class="nowrap"><code class="redtext">string</code></td>
+      <td><code class="bluetext">synced_status_id</code></td>
+      <td>The system id for the status that is synced across the company (read only)</td>
```

Comment:
@elanovasconcelos Would try to shorten this:
"System id for status synced across this company (read only)"